### PR TITLE
Add usage documentation

### PR DESCRIPTION
**Pull request by chrissimpkins**
TODO:

- [ ] update build documentation link https://github.com/source-foundry/alt-hack/commit/11d96c59acfb92392cd763696c2ddb8107c7d222#diff-04c6e90faac2675aa89e2176d2eec7d8R22 when Hack v3.000 is merged to master
- [ ] update build documentation link https://github.com/source-foundry/alt-hack/pull/7/commits/11d96c59acfb92392cd763696c2ddb8107c7d222#diff-04c6e90faac2675aa89e2176d2eec7d8R47 when Hack v3.000 is merged to master

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+The `*.glif` source files are located in the `glyphs` directory of this repository and are organized by Unicode code point. A short string is added to the Unicode code point to describe the type of design modification that was performed to the Hack default style.  Source files are supplied for regular, bold, italic, and bold italic font variants.
+
+You may view images of the alternate designs in the README.md files at the top level of each alternate glyph directory.
+
+
+### How to Build a Custom Version of Hack
+
+### Quickstart Instructions
+
+- Download the [Hack typeface source](https://github.com/source-foundry/Hack/archive/master.zip)
```
Comment (mynetx) [inline README.md]: Add trailing periods to these items.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,59 @@
...
+- Download the `*.glif` source files from the alt-hack repository that you would like to use as replacements in the Hack source
+- Overwrite the existing Hack `*.glif` files for your selected Unicode code points with the alt-hack `*.glif` files (replacements in alt-hack have the same file paths as those in default source)
+- Build new Hack fonts using the simple make based build approach in the [build documentation](#)
```
Comment (mynetx) [inline README.md]: A link is missing?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,59 @@
...
+
+#### Obtain replacement source files
+Download the `*.glif` files in the regular, bold, italic, and bolditalic directories from the alt-hack repository for each replacement that you want to make in the Hack source.  The alternate style source files have the same file path as we use in the Hack source.  The `*.glif` design source files are intended to be used as direct replacements for the existing default source files.
+
+#### Overwrite Existing Source Files
+Copy the alternate glyph source files for the appropriate font variants (i.e. regular, bold, italic, bold italic) from this repository into the corresponding `glyphs` directory of the Hack UFO source for the respective variant.  This should overwrite an existing file in that directory.  Please note that the alternate files for each variant have the same file path. Please confirm that you are overwriting an existing file with a new file that is of the appropriate variant type.
+
+#### Build new fonts
+
+Use the build instructions in the Hack repository to compile new desktop ttf font files and/or web fonts with your custom version of Hack.
```
Comment (mynetx) [inline README.md]: TTF or `*.ttf`.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,59 @@
...
+
+### Post-Modification Licensing
+These changes to the Hack typeface fall within acceptable modification practice under the Hack license.
+
+- It is not necessary to change the font name as a result of these modifications.
+- It is acceptable practice to change the name of the fonts if you would like (e.g. for release to others under a different font name).
+- Please maintain the Hack license with modified versions of the upstream source
```
Comment (mynetx) [inline README.md]: Period?

**Review by mynetx — CHANGES_REQUESTED** [review]

**Comment by chrissimpkins** [comment]
Want to add these changes? You have write access.  Feel free to write to dev branch so reflected in the PR.  We can discuss as changes are made.

**Comment by mynetx** [comment]
@chrissimpkins Yes course. Just not right now, either later today or tomorrow.

**Comment by chrissimpkins** [comment]
@jublo no rush.  We won't merge until we merge Hack dev to master (~Thursday).  Will be working on some doc updates after the Hack merge and if you haven't had a chance by then I will make all of your suggested edits and merge this.  If you get to it before I do feel free to merge to master once v3.000 release of Hack is in master.   These docs assume those changes.

**Pull request merged**